app/src/gateway/gateway.service.ts

```diff
@@ -20,21 +21,21 @@ export class GatewayService
   io: Namespace;
 
   afterInit(): void {
-    console.log("Websocket gateway initialized.");
+    Logger.log("Websocket gateway initialized.");
```

Comment:
Why are we accessing `.log` on `Logger` directly here as opposed to defining an instance of the class like in blocks service before accessing it?

Reply:
+1, whatever pattern/approach we take we should be using the instanced logger